src/app/components/components.component.html

```diff
@@ -13,34 +14,58 @@
                           [sidenavWidth]="(media.registerQuery('gt-xs') | async) ? (miniNav ? '64px' : '257px') : '100%'">
     <div td-sidenav-content layout="column" layout-fill>
       <mat-nav-list flex
-        [tdLayoutManageListClose]="!media.query('gt-sm')" layout="column" layout-align="center center">
-        <a mat-list-item color="accent" [routerLinkActiveOptions]="{exact:true}" [routerLink]="'/chart-types'" routerLinkActive="active">
-          <mat-icon matListIcon [matTooltip]="miniNav ? 'Chart Types' : ''" matTooltipPosition="after">apps</mat-icon>
-          <span *ngIf="!miniNav">Chart Types</span>
+                    [tdLayoutManageListClose]="!media.query('gt-sm')"
+                    layout="column"
+                    layout-align="center center">
+        <a mat-list-item
+            color="accent"
+            [routerLinkActiveOptions]="{exact:true}"
+            [routerLink]="['/components']"
+            routerLinkActive="active">
+          <mat-icon matListIcon
+                    [matTooltip]="miniNav ? 'Overview' : ''"
+                    matTooltipPosition="after">apps</mat-icon>
+          <span *ngIf="!miniNav">Overview</span>
+        </a>
+        <a mat-list-item
+            color="accent"
+            [routerLinkActiveOptions]="{exact:true}"
+            [routerLink]="['/components', 'getting-started']"
+            routerLinkActive="active">
+          <mat-icon matListIcon
+                    [matTooltip]="miniNav ? 'Getting Started' : ''"
+                    matTooltipPosition="after">chrome_reader_mode</mat-icon>
+          <span *ngIf="!miniNav">Getting Started</span>
         </a>
         <mat-divider></mat-divider>
-        <h3 matSubheader *ngIf="!miniNav">Core Charts</h3>
-        <mat-list-item *ngIf="miniNav">
-          <mat-icon matTooltip="Core Charts" matTooltipPosition="after" class="pad-left-xs tc-grey-600">gps_fixed</mat-icon>
-        </mat-list-item>
+        <h3 matSubheader>
+          <span *ngIf="!miniNav">Charts</span>
+          <mat-icon *ngIf="miniNav"
+                    matTooltip="Core Charts"
+                    matTooltipPosition="after"
+                    class="pad-left-xs tc-grey-600">gps_fixed</mat-icon>
```

Comment:
Using `gps_fixed` icon for Core Charts looks strange to me.  What about `collections`? /shrug